data/json/items/armor/belts.json

```diff
@@ -209,17 +210,18 @@
     "covers": [ "TORSO" ],
     "coverage": 20,
     "storage": "1 L",
-    "encumbrance": 10,
+    "encumbrance": 2,
     "material_thickness": 2,
     "use_action": {
       "type": "holster",
-      "holster_prompt": "Attach what to belt loop?",
-      "holster_msg": "You clip your %s to your %s",
-      "multi": 4,
-      "max_volume": "1 L",
-      "max_weight": 1000,
+      "holster_prompt": "Store tool or blade",
+      "holster_msg": "You put your %s in your %s",
+      "multi": 6,
+      "min_volume": "250 ml",
+      "max_volume": "1500 ml",
+      "max_weight": "1500 g",
       "draw_cost": 50,
-      "flags": [ "BELT_CLIP" ]
+      "flags": [ "BELT_CLIP", "SHEATH_KNIFE", "SHEATH_SWORD" ]
```

Comment:
The addition of `SHEATH_SWORD` seems risky, especially combined with increasing volume to 1.5L. What’s the encumbrance like with six two-by-swords or wakizashis attached?

Reply:
32 and 38 encumbrance.

Reply:
Sheathing swords will probably get reverted once KorG gets through the holster refactor, since swords would be too long to fit in the pouch.